specification/hugr.md

```diff
@@ -138,6 +144,15 @@ EdgeKind ::= Hierarchy | Value(Locality, SimpleType) | Static(Locality, ClassicT
 
 Locality ::= Local | Ext | Dom
 ```
+
+Ports always exist on `Value` and `Static` edges; never on `Hierarchy`, `Order `
+or `ControlFlow` edges. Every port has an associated `SimpleType`. The source
+and target ports of an edge have the same `SimpleType`.
+
+A source port with a `ClassicType` may have any number of edges associated with
+it (including zero). A port with a `LinearType`, and a target port of any type,
```

Comment:
maybe worth noting explicitly that this is the encoding of the "value used exactly once"